client/components/Body.js

```diff
@@ -5,13 +5,29 @@
 var React = require('react');
 var CodeMirror = require('./CodeMirror');
 var Analysis = require('./Analysis');
+var $ = require('jquery');
 
 var Body = React.createClass({
+  getCode: function(newCode) {
+    $.ajax({
+      type: 'POST',
+      url: '/parse/:dataType',
+      dataType: 'json',
+      contentType: 'application/json',
+      data: JSON.stringify({data: newCode}),
+      success: function() {
+        console.log('Post request was successful');
+      },
+      error: function(err) {
+        console.error('Ooop! You have a ' + err.status + ' error.');
+      }
+    })
+  },
   render: function() {
```

Comment:
insert a blank line between lines 25 and 26 (separating the getCode and render functions)